### PR TITLE
Visit card personal on entering area

### PR DESCRIPTION
**Pull request by matthintosh**
On entering in a personal area, a user will see the visit card of the area owner.

### CODE REVIEW

File: libs/map-editor/src/types.ts [hunk 1]
```diff
@@ -131,7 +131,7 @@ export const PersonalAreaPropertyData = PropertyBase.extend({
     type: z.literal("personalAreaPropertyData"),
     accessClaimMode: PersonalAreaAccessClaimMode,
     allowedTags: z.array(z.string()).default([]),
-    ownerId: z.string().default(""),
+    ownerId: z.string().nullable(),
```
Comment (gparant) [inline libs/map-editor/src/types.ts:134]: use optional, by default the proto message serialiser can't take null option

File: libs/messages/src/JsonMessages/MemberData.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ export const MemberData = z.object({
   id: z.string(),
   name: z.string().nullable(),
   email: z.string().nullable(),
+  visitCardUrl: z.string().nullable(),
```
Comment (gparant) [inline libs/messages/src/JsonMessages/MemberData.ts:7]: same

File: play/src/pusher/services/AdminApi.ts [hunk 3]
```diff
@@ -1022,6 +1022,34 @@ class AdminApi implements AdminInterface {
         });
         return response.data ? response.data : [];
     }
+
+    /**
+     * @openapi
+     * /members:
```
Comment (gparant) [inline play/src/pusher/services/AdminApi.ts:1028]: that is correct ?
If we search by `memberUUID`, we could use "/members/{uuid}"?

Comment (matthintosh) [reply play/src/pusher/services/AdminApi.ts]: I added the missing parameter for openApi documentation.

**Pull request merged**